src/components/Tooltip/index.js

```diff
@@ -0,0 +1,57 @@
+import React, {PropTypes} from 'react'
+
+const passStrengthValues = [10, 20, 25, 30, 33, 34, 40, 50, 60, 70, 75, 80, 90, 100]
+
+const Tooltip = ({children}) => {
+  const ttClasses = 'relative db br2 w-100 mt4 pa3 sans-serif navy bg-white eh-input-tooltip'  
+  return (
+    <div className={ttClasses}>
+      {children}
+    </div>
+  )
+}
+Tooltip.propTypes = {
+  children: React.PropTypes.node.isRequired
+}
+
+/**
+ * Password-specific Tooltip
```

Comment:
@tayiorbeii there is room for variants of components. The issue with specificity would be application specific. I think password validity/strength logic is definitely out of place in our base tooltip.

I'm fairly anti comments in code, so any time I see them it triggers a "why?" :)